src/views/Feed/FeedExtra.js

```diff
@@ -0,0 +1,62 @@
+import _ from 'lodash'
+import cx from 'classnames'
+import React, { PropTypes } from 'react'
+
+import META from '../../utils/Meta'
+import { customPropTypes, getUnhandledProps, useKeyOnly } from '../../utils/propUtils'
+
+function FeedExtra(props) {
+  const { children, className, images, text } = props
+  const classes = cx(
+    className,
+    useKeyOnly(images, 'images'),
+    useKeyOnly(text, 'text'),
+    'extra'
+  )
+  const rest = getUnhandledProps(FeedExtra, props)
+
+  if (Array.isArray(images)) {
+    const imagesJSX = images.map((image, i) => {
+      return _.isString(image) ? <img key={i} src={image} /> : image
```

Comment:
>@levithomason It seems that you like antipattern with key more than I 😆

I must admit, I once took it very lightly :S

>- allow pass only string, not nodes;
>- use pair of index and src.

There is a chance that the strings might be the same, for instance a placeholder image for multiple images.  Also, adding index into the string will affectively do the same thing as having an index only, since the key changes when the child order changes.  We would need to accept objects in order to accept a `key`: `images={[{key: 'foo', src: '...'}]}`.

I think for now, we allow a string only and fix the duplicate image src string edge case if it arises.